--- a/node_modules/.pnpm/gray-matter@4.0.3/node_modules/gray-matter/index.js
+++ b/node_modules/.pnpm/gray-matter@4.0.3/node_modules/gray-matter/index.js
@@ -8,7 +8,7 @@
 const excerpt = require('./lib/excerpt');
 const engines = require('./lib/engines');
 const toFile = require('./lib/to-file');
 const parse = require('./lib/parse');
-const utils = require('./lib/Utils');
+const utils = require('./lib/utils');

 /**
  * Takes a string or object with `content` property, extracts